### ISSUE TITLE
Create 2D training environment for gym

### ISSUE THREAD

**Issue by michaelwm** [uwreact/uwreact_robot#44]
# 🚀 Feature Request

Part of #42. Depends on #43. We must create our 2D training environment for `gym`, because `keras-rl` works best with `gym` environments, and it is our chosen deep learning library. The focus is on 2019: creating a generic tool for this is not essential, but it will be very beneficial for future years, and the task of a separate issue.

**Comment by michaelwm** [comment]
Some initial resources on creating a custom `gym` environment for `keras-rl`:
1. https://github.com/keras-rl/keras-rl/issues/38
2. https://medium.com/@apoddar573/making-your-own-custom-environment-in-gym-c3b65ff8cdaa

**Comment by michaelwm** [comment]
Important attributes to cover:
1. Important positions (in front of rocket, etc...)
2. State at positions (cargo, hatches, etc...)
3. Line-of-sight transitions between positions (movement)
4. Action transitions at positions (pick-up cargo, score cargo, etc...)

**Comment by michaelwm** [comment]
Please suggest more important attributes to cover in our first iteration as discussed in the first meeting, in case I missed anything.

**Comment by michaelwm** [comment]
Switch `keras-rl` to `pytorch`

**Comment by michaelwm** [comment]
Further resources I am using to develop the training environment:
1. https://stackoverflow.com/questions/45068568/is-it-possible-to-create-a-new-gym-environment-in-openai
2. https://gym.openai.com/docs/#environments
3. https://github.com/tambetm/gym-minecraft
